api-tester/build.gradle.kts

```diff
@@ -13,6 +13,7 @@ android {
         create("defaults") {
             dimension = "apis"
             isDefault = true
+            missingDimensionStrategy("billingclient", "bc8")
```

Comment:
This flavor dimension doesn't AND shouldn't modify the public API, so I don't think it's worth adding api tests for the bc7 flavor.